patch_parser_test.py

```diff
@@ -1,11 +1,16 @@
 import unittest
 from patch_parser import parse_comments
-from archive_converter import find_thread, generate_email_from_file
+from archive_converter import ArchiveMessageIndex, generate_email_from_file
+from message_dao import MessageDao
 
 class PatchParserTest(unittest.TestCase):
 
     def test_parse_comments_for_single_email_thread(self):
-        patchset = parse_comments(find_thread('PATCH v2 1/2'))
+        archive_index = ArchiveMessageIndex(MessageDao())
```

Comment:
should we have at least one test case verify the return value of `update()`?

Reply:
do you think I should hard code in the order? Like, I can expect patch6 to be the first value, and thread_patch0 to be the second.